src/main/java/de/hbz/lobid/helper/CreateWikidataNwbibMaps.java

```diff
@@ -0,0 +1,81 @@
+/* Copyright 2019 hbz, Pascal Christoph. Licensed under the EPL 2.0*/
+
+package de.hbz.lobid.helper;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.hp.hpl.jena.rdf.model.Model;
+import com.hp.hpl.jena.rdf.model.ModelFactory;
+import com.hp.hpl.jena.rdf.model.Property;
+import com.hp.hpl.jena.rdf.model.ResIterator;
+import com.hp.hpl.jena.rdf.model.Resource;
+
+/**
+ * Creates nwbib maps used by morph.
+ * 
+ * @author Pascal Christoph (dr0i)
+ *
+ */
+public class CreateWikidataNwbibMaps {
+	private static final Logger LOG =
+			LogManager.getLogger(CreateWikidataNwbibMaps.class);
+	private static final String WARN =
+			"will not renew the map but going with the old one";
+	private static Model model = ModelFactory.createDefaultModel();
```

Comment:
Since it's not reassigned, I think the `model` should be a constant too (--> `private static final`, and uppercase `MODEL`).